### PR TITLE
Update token server docs to generalize outside of GD

### PR DESCRIPTION
**Pull request by juliexxia**

### CODE REVIEW

File: token_server/README.md [hunk 1]
```diff
@@ -4,104 +4,108 @@ This is an optional component for the mapping page, that allows any user to view
...
-   created above.
+*   You should not normally need to do this unless you had to switch Google
+    accounts, as described in [INITIAL_SETUP.md](../docs/INITIAL_SETUP.md).
+*   If you have already followed the instructions at
+    [INITIAL_SETUP.md](../docs/INITIAL_SETUP.md) to set up the mapping page in
+    the first place, most of the work needed here is already done. The only
+    additional work is to create a service account that is whitelisted for
+    EarthEngine access (currently `earthengine-token-provider`) and give it the
+    "Service Account Token Creator" role in the
+    [IAM page](https://console.developers.google.com/iam-admin/iam?project=mapping-crisis).
```
Comment (janakdr) [inline token_server/README.md]: they'll need to change the project name here.

File: token_server/README.md [hunk 2]
```diff
@@ -4,104 +4,108 @@ This is an optional component for the mapping page, that allows any user to view
...
-     -  [Give access only to this secret](https://docs.aws.amazon.com/secretsmanager/latest/userguide/auth-and-access_identity-based-policies.html#permissions_grant-limited-resources).
-     -  After deploying the app, in the Elastic Beanstalk console, go to
-        Configuration > Security > Modify and choose the IAM instance profile
-        you created above.
+    -   Make sure the region you created the secret in matches the region
+        specified in `AWS_REGION` in
+        [aws_get_credentials.js](./aws_get_credentials.js)!
+
+    To make the secret readable by the server, you will have to have the server
+    run as a user with permissions to read the secret: -
```
Comment (janakdr) [inline token_server/README.md]: what happened here? and below

Comment (juliexxia) [reply token_server/README.md]: Fixed I think

File: token_server/README.md [hunk 3]
```diff
@@ -4,104 +4,109 @@ This is an optional component for the mapping page, that allows any user to view
...
-   [these instructions](https://aws.amazon.com/getting-started/tutorials/deploy-app-command-line-elastic-beanstalk/).
-   Since the project already exists on Amazon Elastic Beanstalk, you just need
-   to run `eb deploy` after running `eb init` in order to log in.
+*   Deploy using
+    [these instructions](https://aws.amazon.com/getting-started/tutorials/deploy-app-command-line-elastic-beanstalk/).
+    Since the project already exists on Amazon Elastic Beanstalk, you just need
+    to run `eb deploy` after running `eb init` in order to log in.
 
-*  The server URL will be entered here once we have GiveDirectly's account.
+*   The server URL will be entered here once we have GiveDirectly's account.
```
Comment (janakdr) [inline token_server/README.md]: maybe put this in parentheses or something so it's clear that it's not an instruction for users? To be honest, my guess is we'll never actually use this section.

Comment (juliexxia) [reply token_server/README.md]: Updated to I think what you meant

**Pull request merged**